client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/BackfillOperator.kt

```diff
@@ -295,6 +297,15 @@ internal class BackfillOperator<E : DbEntity<E>, Pkey : Any> internal constructo
     return RunBatchResponse.Builder().build()
   }
 
+  private fun Backfill<*, *>.getPrimaryKeyPath(queryRoot: Root<*>): Path<Number> {
```

Comment:
I think the place to fix this is in Misk? @swankjesse Thoughts?

Reply:
@zhxnlai I'm not sure we want that other PR either?

Reply:
@mpawliszyn I think the fix should be in backfila because CriteriaBuilder is a JPA API: https://www.objectdb.com/java/jpa/query/criteria